index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<head>
+	<title>Ecxercise 3</title>
```

Comment:
slight misspelling of the word exercise, also I'm assuming you might want to call this page Portfolio anyway right? or something to that effect

also, the indentation here is 1 tab extra, you probably need it to be on the same level as the `<link>` tags below